app/views/users/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<% provide(:title, t( "list_attr.title" )) %>
+<h1><%= t "list_attr.title" %></h1>
+<%= will_paginate %>
+<ul class="users">
+  <% @users.each do |user| %>
+  <%= render @users %>
```

Comment:
Em đọc thêm về render partial trong rails nhé
https://viblo.asia/p/render-partials-in-rails-ZjleadVEGqJ

Đã dùng  <%= render @users %> rồi thì không cần each nữa

Reply:
Dạ anh.

Reply:
Đẫ fix ạ.